cairo-contracts/packages/clients/src/cometbft/errors.cairo

```diff
@@ -4,7 +4,7 @@ pub mod CometErrors {
     pub const INVALID_CLIENT_STATE: felt252 = 'ICS07: invalid client state';
     pub const INVALID_CONSENSUS_STATE: felt252 = 'ICS07: invalid consensus state';
     pub const INVALID_HEADER: felt252 = 'ICS07: invalid header';
-    pub const INVALID_HEADER_TIMESTAMP: felt252 = 'ICS07: invalid header timestamp';
+    pub const HEADER_TIMESTAMP_FROM_FUTURE: felt252 = 'ICS07: header ts from future';
```

Comment:
I’ve been trying to stick to an implicit convention for naming errors, kind of like what we did in `ibc-rs`, just to keep things consistent. Does the following make sense to you?

```suggestion
    pub const INVALID_HEADER_FROM_FUTURE: felt252 = 'ICS07: inv header from future';
```